sycl/CMakeLists.txt

```diff
@@ -6,6 +6,16 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 option(SYCL_ENABLE_WERROR "Treat all warnings as errors in SYCL project" OFF)
+option(SYCL_ADD_DEV_VERSION_POSTFIX "Adds -V postfix to version string" ON)
+
+set(SYCL_MAJOR_VERSION 0)
+set(SYCL_MINOR_VERSION 1)
+set(SYCL_PATCH_VERSION 0)
+set(SYCL_DEV_ABI_VERSION 0)
```

Comment:
These numbers should be manually incremented every time we make some breaking or non-breaking changes?

Reply:
Only DEV_ABI version must be incremented, and only when breaking change happens. The rest of the numbers is changed once per release.

Reply:
Thanks for clarification.